cirq-google/cirq_google/transformers/target_gatesets/google_cz_gateset_test.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+import cirq
+import cirq_google
+
+
+_qa, _qb = cirq.NamedQubit('a'), cirq.NamedQubit('b')
+
+
+@pytest.mark.parametrize(
+    'before',
+    [
+        cirq.Circuit(cirq.Z(_qa) ** 0.5, cirq.CZ(_qa, _qb)),
+        cirq.Circuit(cirq.Z(_qa).with_tags(cirq_google.PhysicalZTag()) ** 0.5, cirq.CZ(_qa, _qb)),
+        cirq.Circuit(cirq.PhasedXPowGate(phase_exponent=0.125).on(_qa), cirq.CZ(_qa, _qb)),
+    ],
+)
+def test_eject_paulis_disabled(before):
+    after = cirq.optimize_for_target_gateset(
+        before,
+        gateset=cirq_google.GoogleCZTargetGateset(
+            additional_gates=[
+                cirq.GateFamily(cirq.ZPowGate, tags_to_ignore=[cirq_google.PhysicalZTag()]),
+                cirq.GateFamily(cirq.ZPowGate, tags_to_accept=[cirq_google.PhysicalZTag()]),
```

Comment:
Isn't including both of these gate families is equivalent to including the type gate family for `cirq.ZPowGate` ?

Reply:
That's true. My preference is to keep it the way it is, to mimic the GateFamily entries in GridDevice, but either way works.

Reply:
I think including both in a test is confusing. It would be better to have different tests; by including one of each (maybe controlled by a flag)? 

Also; do you think we should have a default value for `additional_gates` in the Google target gateset? What's the default for a google device? Do we accept both virtual and physical zs by default?

Reply:
Synced offline. Will stay with an empty default. Tests are now split.